biogeochem/EDLoggingMortalityMod.F90

```diff
@@ -168,41 +184,88 @@ end subroutine IsItLoggingTime
    ! ======================================================================================
 
    subroutine LoggingMortality_frac( pft_i, dbh, canopy_layer, lmort_direct, &
-                                     lmort_collateral,lmort_infra, l_degrad )
+                                     lmort_collateral,lmort_infra, l_degrad, &
+                                     hlm_harvest, hlm_harvest_catnames, &
+                                     use_history, secondary_age, &
+                                     frac_site_primary)
 
       ! Arguments
       integer,  intent(in)  :: pft_i            ! pft index 
       real(r8), intent(in)  :: dbh              ! diameter at breast height (cm)
       integer,  intent(in)  :: canopy_layer     ! canopy layer of this cohort
+      real(r8), intent(in) :: hlm_harvest(:)       ! annual harvest rate per hlm category
+      character(len=64), intent(in) :: hlm_harvest_catnames(:) ! names of hlm harvest categories
+      integer, intent(in) :: use_history        ! patch level anthro_disturbance_label
+      real(r8), intent(in) :: secondary_age     ! patch level age_since_anthro_disturbance
       real(r8), intent(out) :: lmort_direct     ! direct (harvestable) mortality fraction
       real(r8), intent(out) :: lmort_collateral ! collateral damage mortality fraction
       real(r8), intent(out) :: lmort_infra      ! infrastructure mortality fraction
       real(r8), intent(out) :: l_degrad         ! fraction of trees that are not killed
                                                 ! but suffer from forest degradation (i.e. they
                                                 ! are moved to newly-anthro-disturbed secondary
                                                 ! forest patch)
+      real(r8), intent(in) :: frac_site_primary
 
-      ! Parameters
-      real(r8), parameter   :: adjustment = 1.0 ! adjustment for mortality rates
- 
+      ! Local variables
+      real(r8) :: harvest_rate ! the final harvest rate to apply to this cohort today
+
+      ! todo: probably lower the dbhmin default value to 30 cm
+      ! todo: change the default logging_event_code to 1 september (-244)
+      ! todo: change the default logging_direct_frac to 1.0 for cmip inputs
+      ! todo: check outputs against the LUH2 carbon data
+      ! todo: eventually set up distinct harvest practices, each with a set of input paramaeters
+      ! todo: implement harvested carbon inputs
+      
       if (logging_time) then 
```

Comment:
we discussed this and agreed that the code is correct as-is, since the bc_in%harvest_rates are only really updated yearly and the point of the IsItLoggingTime is to downscale those yearly average rates to some fates-specified frequency (on one specific day per year, or monthly, etc).